nntrainer/layers/common_properties.cpp

```diff
@@ -242,4 +269,46 @@ str_converter<props::connection_prop_tag, props::ConnectionSpec>::from_string(
   return props::ConnectionSpec({n});
 }
 
+template <>
+std::string
+str_converter<props::tensors_shape_prop_tag, props::TensorsSpec>::to_string(
+  const props::TensorsSpec &value) {
```

Comment:
can this function be called with value where value.getTensorShape().size() == 0?

Reply:
if it is 0, then it will give empty string.

Reply:
aah, then its fine, thanks.